config/routes.rb

```diff
@@ -18,7 +18,7 @@
   resources :statistics, only: [:average_marks_groups] do
     collection do
       get 'average_marks_groups', to: :average_marks_groups
-      get 'marks_groups_quizzes', to: :marks_groups_quizzes
+      post 'marks_groups_quizzes', to: :marks_groups_quizzes
```

Comment:
pentru ca am schimbat putin structura si acum nu mai dam toate sesiunile pt toate grupurile, ci specifica frontul ce grup in post request